modules/data-table/main/index.coffee

```diff
@@ -200,6 +488,10 @@ class DataTable extends hx.EventEmitter
   displayMode: option('displayMode')
   feed: option('feed')
   filter: option('filter')
+  advancedSearch: option('advancedSearch')
```

Comment:
advancedSearch is independent of filter - this lets us switch between the two in a much nicer fashion without complicating the filter code